src/popup.html

```diff
@@ -6,6 +6,9 @@
     <title>Popup</title>
 
     <link rel="stylesheet" href="popup.css">
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
We should manually include `bootstrap.min.css` with the package. Why? Because chrome extensions have stick permission rules. Users would have to give the extension permission to access this url.